android-components/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -335,6 +335,14 @@ class BrowserToolbar @JvmOverloads constructor(
         edit.removeEditActionEnd(action)
     }
 
+    override fun hideMenuButton() {
+        display.hideMenuButton()
+    }
+
+    override fun showMenuButton() {
+        display.showMenuButton()
+    }
```

Comment:
Are we able to add any tests for `BrowserToolbar`, `DisplayToolbar`, `MenuButton` and `ToolbarView`?

Reply:
Ah yes, I'll add tests and a changelog entry for the A-C changes

Reply:
tests and changelog entry added